internal/kafka/internal/handlers/kafka_instance_types.go

```diff
@@ -0,0 +1,56 @@
+package handlers
+
+import (
+	"net/http"
+
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/api/public"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/presenters"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/services"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/errors"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/handlers"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/logger"
+	"github.com/gorilla/mux"
+)
+
+type supportedKafkaInstanceTypesHandler struct {
+	service services.SupportedKafkaInstanceTypesService
+}
+
+func NewSupportedKafkaInstanceTypesHandler(service services.SupportedKafkaInstanceTypesService) *supportedKafkaInstanceTypesHandler {
+	return &supportedKafkaInstanceTypesHandler{
+		service: service,
+	}
+}
+
+func (h supportedKafkaInstanceTypesHandler) ListSupportedKafkaInstanceTypes(w http.ResponseWriter, r *http.Request) {
```

Comment:
Can we change this to a pointer to avoid copying the `supportedKafkaInstanceTypesHandler` at every call to `ListSupportedKafkaInstanceTypes`?
```suggestion
func (h *supportedKafkaInstanceTypesHandler) ListSupportedKafkaInstanceTypes(w http.ResponseWriter, r *http.Request) {
```

Reply:
Thanks I'll update this